src/Component/Footer/footcss.css

```diff
@@ -15,19 +15,21 @@ footer {
   display: flex;
   justify-content: center;
   align-items: center;
-  padding-bottom: 3vh;
+  height: 117px;
+  padding-bottom: 30px;
 }
 
 .inslog {
+  /* padding-left: 10px; */
```

Comment:
remove unnecessary comments

Reply:
done

Reply:
you have to push the code too

Also remove all unnecessary logs and comments from both files